Modules/commands/_feature.py

```diff
@@ -0,0 +1,145 @@
+"""
+_feature.py - {summery}
+
+{long description}
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+
+from logging import getLogger
```

Comment:
I don't recall where i read this, but for some reason i thought it would be better to import a specific name from another module, rather than the whole module itself. I only needed the one function from it therefore i only imported that name.

reverted now to the `import all the things` approach.

(makes no functional difference)

Reply:
This still looks like its missing the Logger import?

Reply:
So, I believe we chatted about this so this is more for the record.

There is no functional difference between the two, and this point is purely stylistic / opinion.

IMHO, since we are only importing a single name from the module its more appropriate to do a `from foo import bar`, as we only retreive the target name.

Of course, this argument is purely on semantics so I can change it if that is what we want.

Reply:
Resolved in IRC.